src/access-api.ts

```diff
@@ -538,7 +538,7 @@ export class AccessApi extends EventEmitter {
     }
 
     // Unlocking only works on hubs.
-    if(device.device_type !== "UAH") {
+    if(device.device_type !== "UAH" && device.device_type !== "UA-ULTRA") {
```

Comment:
Was debugging an issue with my setup today and discovered that my device type is "UAH-DOOR" - would be ideal if you're able to add "&& device.device_type != "UAH-DOOR"" to this logic. Can also initiate another PR if you'd like!